toxav/BUILD.bazel

```diff
@@ -29,99 +29,46 @@ cc_test(
     ],
 )
 
-cc_library(
-    name = "ring_buffer_srcs",
-    hdrs = [
-        "ring_buffer.c",
-        "ring_buffer.h",
-    ],
-    visibility = ["//c-toxcore/testing:__pkg__"],
-    deps = ["//c-toxcore/toxcore:ccompat"],
-)
-
-cc_library(
-    name = "bwcontroller",
-    srcs = ["bwcontroller.c"],
-    hdrs = ["bwcontroller.h"],
-    deps = [
-        ":ring_buffer",
-        "//c-toxcore/toxcore",
-        "//c-toxcore/toxcore:Messenger",
-    ],
-)
-
-cc_library(
-    name = "rtp",
-    srcs = ["rtp.c"],
-    hdrs = ["rtp.h"],
-    deps = [":bwcontroller"],
-)
-
 cc_test(
     name = "rtp_test",
     size = "small",
     srcs = ["rtp_test.cc"],
     deps = [
-        ":rtp",
+        ":toxav",
         "//c-toxcore/toxcore:crypto_core",
         "@com_google_googletest//:gtest_main",
     ],
 )
 
-cc_library(
-    name = "audio",
-    srcs = ["audio.c"],
-    hdrs = ["audio.h"],
-    deps = [
-        ":public",
-        ":rtp",
-        "//c-toxcore/toxcore:network",
-        "@opus",
-    ],
-)
-
-cc_library(
-    name = "video",
-    srcs = [
-        "msi.c",
-        "video.c",
-    ],
-    hdrs = [
-        "msi.h",
-        "video.h",
-    ],
-    deps = [
-        ":audio",
-        ":public",
-        "//c-toxcore/toxcore:network",
-        "@libvpx",
-    ],
-)
-
-cc_library(
-    name = "groupav",
-    srcs = ["groupav.c"],
-    hdrs = ["groupav.h"],
-    deps = [
-        "//c-toxcore/toxcore",
-        "@opus",
-    ],
-)
-
 cc_library(
     name = "toxav",
     srcs = [
+        "audio.c",
```

Comment:
What's the plan around breaking this big dependency cycle? There used to be some form of layering here.

Reply:
i have no idea, last time this was the compromise. maybe a future cleanup?